packages/opentelemetry-api/src/api/propagation.ts

```diff
@@ -53,19 +54,29 @@ export class PropagationAPI {
    * Inject context into a carrier to be propagated inter-process
    *
    * @param carrier carrier to inject context into
+   * @param setter Function used to set values on the carrier
    * @param context Context carrying tracing data to inject. Defaults to the currently active context.
    */
-  public inject(carrier: Carrier, context = contextApi.active()): void {
-    return this._propagator.inject(context, carrier);
+  public inject<Carrier = any>(
```

Comment:
would it be better to put `unknown` instead of `any` if we need to modify those functions later ?

Reply:
1c06785e5e3d42468f222661f42c29242259364c what about this?